post_installation_configuration/cluster-capabilities.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="cluster-capabilities"]
+= Cluster capabilities
+include::_attributes/common-attributes.adoc[]
+:context: cluster-capabilities
+
+toc::[]
+
+The cluster capabilities provide an option for cluster administrators to select or deselect one or more optional components at any time post installation.
+
+[NOTE]
+====
+If cluster capability has been enabled before, it cannot be disabled.
+====
+
+include::modules/viewing-cluster-capabilities.adoc[leveloffset=+1]
+
+include::modules/managing-cluster-capabilites.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+The following are the capabilities defined in `v4.11`:
+
+* Enables xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install[metal3 deployment]
```

Comment:
If changing it to the capability name `baremetal`, it would be easy to read and understand for me:)